roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,22 +15,37 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
     docker_image_base = {
         "origin": {
             "repo": "openshift",
             "image": "origin",
         },
+        "online": {
```

Comment:
Both fall into the `else` clause... could have been an oversight? Anything that doesn't fall into the explicit `if` clauses end up treated as "origin"...

When I was looking for docs about deployment types, I saw "online" in the list of known values, but it is not documented in DEPLOYMENT_TYPES.md.

Would be good to investigate how is it used and what is the intention of setting `deployment_type=online` in the inventory file.